### DISCUSSION TITLE
What to do when the user imports an outdated blob file with dexie-export-import

### DISCUSSION THREAD

**Discussion by laukaichung** [dexie/Dexie.js#2051] [Q&A] [answered]
Updated: I just found `transform()`. I guess it is the way to do it.

----------------------------

Hello,
How would you handle a scenario where a user imports a blob file that is outdated or an older version? Is there a way to modify the data to fit the latest schema? l couldn't find something like an `onPut(row)` callback in ImportOption. Would you just use `filter()` option to exclude the outdated data?

**Comment by dfahlander** [comment, marked as answer]
> Updated: I just found `transform()`. I guess it is the way to do it.

Yes, transform can be used to modify the data before import. I think the docs might not have been updated with it.
